garden-backend-service/compose.yaml

```
@@ -0,0 +1,30 @@
+services:
+  dev-api:
+    build:
+      context: .
+    ports:
+      - "5500:80"
+    volumes:
+      - ./src:/app/src
+      - ./tests:/app/tests
+      - ./.env:/app/.env
+    entrypoint: bash -c "uvicorn src.main:app --host 0.0.0.0 --port 80 --reload"
+    depends_on:
+      dev-db:
+        condition: service_healthy
+
+  dev-db:
+    hostname: dev-db
+    image: postgres:latest
```

```suggestion
    image: postgres:16
```
afaik this is currently the same as `:latest` but the actual dbs are using postgres 16 so might as well pin it here